src/macaron/output_reporter/results.py

```diff
@@ -62,7 +62,10 @@ class DepSummary(TypedDict):
 
 
 RecordNode = TypeVar("RecordNode", bound="Record")
-"""This binds type ``RecordNode`` to ``Record`` and any of its subclasses."""
+# This code documentation is commented out because newer Sphinx version (^=6.1.0) has issues processing
+# it.
```

Comment:
```suggestion
# The documentation below for `TypeVar` is commented out due to a breaking 
# change in Sphinx version (^=6.1.0).
```

Reply:
Fix in https://github.com/oracle-samples/macaron/pull/59/commits/5551cbc784af5e36339580c7a1851f5e92cbb109